src/js/player.js

```diff
@@ -300,7 +300,17 @@ class Player extends Component {
     if (tag.parentNode) {
       tag.parentNode.insertBefore(el, tag);
     }
-    Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    //Dom.insertElFirst(tag, el); // Breaks iPhone, fixed in HTML5 setup.
+
+    // append the DOM el function to make use of our component.addChild method
+    // to properly be injected into the children Array
+    tag.el = function () {
```

Comment:
understood @gkatsev - just not sure where this came from. I have a feeling it was @heff during his original input into this branch. Considering the `player.test.js` was also updated to support this. If I go with your suggestion and undo this, it breaks the unit test.